plugins/chat/spec/system/page_objects/chat/chat_channel.rb

```diff
@@ -32,6 +34,19 @@ def expand_message_actions(message)
         click_more_buttons(message)
       end
 
+      def expand_message_actions_mobile(message, delay: 2)
+        message_by_id(message.id).click(delay: delay)
+      end
+
+      def click_message_action_mobile(message, message_action)
```

Comment:
Crazy that we need this...I struggled with it not clicking the mobile buttons properly until I saw your change in Roman's PR about this for selecting messages

Reply:
yep :/ I will extract it into a proper helper at some point. Long press is not a very usual action in web apps.

Reply:
It's so weird, it just decides itself to click on completely random stuff in the mobile menu even when it's been open for a couple of seconds. Nuts!